Create How-to-check-for-CGNAT.md (#118)
BeamMP/Docs: docs/en/FAQ/How-to-check-for-CGNAT.md

How to check for CGNAT?
## Issue

All Firewall and Port forwarding rules are 100% correct, yet nobody can join your home-hosted Server!

If you have connection problems and you are using a Hosting-Service, contact them for assistance. If you want to use a VPS, take a look at this
[list of Partnered hosting services](https://docs.beammp.com/server/create-a-server/#_1:~:text=our%20partnered%20Services!-,Paid%20Services%3A,-Horizon%20Hosting) (Server setup documentation)

# How to check for CGNAT?

## Method 1:

> Open a cmd, run ``tracert -4 beammp.com``. This will output a series of network hops. Wait for the operation to finish (may take up to 30 hops). Check the first few IP addresses after the IP of your Router/Modem/Gateway.

Reviewer: ```suggestion
Open a command prompt, run ``tracert -4 beammp.com``. This will output a series of network hops. Wait for the operation to finish (may take up to 30 hops). Check the first few IP addresses after the IP of your Router/Modem/Gateway.
```

> If multiple IP addresses within the range of 100.64.x.x-100.127.x.x appear after the Gateway/Modem/Router IP (top IP), you are most likely behind a CGNAT.

Reviewer: Can be removed, line 15 says the same but better formatting

Author: Which line exactly?

If multiple IP addresses within the range of ``100.64.x.x``-``100.127.x.x`` or ``10.xx.xx.xx`` appear after the first hop, you are most likely behind a CGNAT.
The first hop is your Router/Modem/Gateway and differs between Devices.

## Method 2:

> Find out the WAN IP on your routers interface. Compare it to the IP posted on e.g. https://whatsmyip.org . If they are NOT the same, you are behind a CGNAT.

Reviewer: ```suggestion
Find out the WAN IP on your routers interface by looking it up in the routers setup website. Compare it to the IP posted on e.g. https://whatsmyip.org . If they are NOT the same, you are behind a CGNAT.
```

Author: I have altered this suggestion. See recent commit.

Author: @O1LER

## Method 3/Solution:
Call your Internet Service Provider for assistance.

> Depending on your ISP, they might not offer dedicated *dynamic* IP adresses (a static IP is not necessary).

Reviewer: ```suggestion
Depending on your ISP, they might not offer a dedicated *dynamic* IP adresses (a static IP is not necessary) or offer it as a paid option.
```

Author: @O1LER
